rootex/framework/entity.cpp

```diff
@@ -49,6 +64,14 @@ JSON::json Entity::getJSON() const
 	{
 		j["Components"][component->getName()] = component->getJSON();
 	}
+	if (m_Script)
+	{
+		j["Entity"]["script"] = m_Script->getJSON();
+	}
+	else
+	{
+		j["Entity"]["script"] = {};
```

Comment:
Set to `nullptr`, so that it appears as `script: null`, instead of `script: {}` in the JSON